tests/executor/indexer/test_elastic_in_flow.py

```diff
@@ -163,7 +163,7 @@ def test_search_with_filtering(self, metas, setup_service_running, flow):
             on='/search',
             inputs=docs_query,
             return_results=True,
-            parameters={'filter': {'tags__price': {'$lt': 50.0}}},
+            parameters={'filter': {'tags__price': {'lt': 50.0}}},
```

Comment:
```suggestion
            parameters={'filter': {'price': {'lt': 50.0}}},
```
test cases should replicate the values we expect

Reply:
Actually, we do expect this format for the filter parameters arriving at the indexer. The BFF will modify an incoming filter that looks like this: `'filter': {'price': {'lt': 50.0}}` to -> `'filter': {'tags__price': {'lt': 50.0}}`

Reply:
The `tags__` prefix helps the es query building to access the correct filter fields found in the tags.